#48 boomerang

H3G0A opened the pull request.

H3G0A committed:
#48 - feat: Agregado prefab de boomerang y su comportamiento
El boomerang sale disparado hacia delante y tras un tiempo vuelve al jugador. Atraviesa terreno solo cuando vuelve y atraviesa enemigos.

H3G0A committed:
#48 - refactor: creado prefab del jugador

H3G0A committed:
#48 - fix: Añadido prefab de jugador a la escena también

H3G0A committed:
Merge branch 'develop' into #48-boomerang

josaloroc reviewed (approved):
Boomerang looks cool!

H3G0A committed:
#48 - merge: merge con rama develop (walljump)
He cambiado valores de wallJumpTime y wallSlideSpeed y también cómo se calcula la velocidad de WallSlideSpeed
